js/service/davservice.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @author Christoph Wurst <christoph@winzerhof-wurst.at>
+ *
+ * ownCloud - Mail
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+define(function(require) {
+	'use strict';
+
+	var dav = require('davclient');
+	var OC = require('OC');
+	var Radio = require('radio');
+
+	Radio.dav.reply('calendars', getUserCalendars);
+
+	var client = new dav.Client({
+		baseUrl: OC.linkToRemote('dav/calendars'),
+		xmlNamespaces: {
+			'DAV:': 'd',
+			'urn:ietf:params:xml:ns:caldav': 'c',
+			'http://apple.com/ns/ical/': 'aapl',
+			'http://owncloud.org/ns': 'oc',
+			'http://calendarserver.org/ns/': 'cs'
+		}
+	});
+	var props = [
+		'{' + dav.NS_DAV + '}displayname',
```

Comment:
Those are only defined in calendar: https://github.com/owncloud/calendar/blob/8e32fee4f2bf1346e987c23d2d0ab05cebcefd40/js/app/service/davclient.js#L39-L43